certified-connectors/Business Assist/apiDefinition.swagger.json

```diff
@@ -672,14 +672,14 @@
       "authorizationUrl": "https://login.windows.net/common/oauth2/authorize",
       "tokenUrl": "https://login.windows.net/common/oauth2/authorize",
       "scopes": {
-        "https://businessassist.microsoft.com/BusinessAssist.Access": "https://businessassist.microsoft.com/BusinessAssist.Access"
+        "https://business-assist.microsoft.com/.default": "https://business-assist.microsoft.com/.default"
```

Comment:
Is `business-assist` correct instead of `businessassist`?

Reply:
Yes this is correct.